### PR TITLE
[LPS-123675]- Fix assertion value none outside none to outside none none

### PR DESCRIPTION
**Pull request by PablitoBonito**
**JIRA Ticket:** 
https://issues.liferay.com/browse/LRQA-63100 
https://issues.liferay.com/browse/LRQA-63099
**Description:**
Fix poshi error java.lang.Exception: CSS Value outside none none does not match none outside none 
by adjusting assertions value from "none outside none" to "outside none none". 

To test enough to run Poshi test from the ticket's title and see if it paseed.

### CODE REVIEW

**Comment by manoelcyreno** [comment]
ci:test:sf

**Comment by manoelcyreno** [comment]
ci:test:relevant

**Comment by manoelcyreno** [comment]
ci:test:frontend

**Comment by manoelcyreno** [comment]
@PablitoBonito , remember to edit your description to tell how to run (how to test) it...

**Review by john-co — CHANGES_REQUESTED** [review]
@PablitoBonito ,
You should split up into two commits. We don't put two tickets in the same message.

Also your title and commit messages are completely opposite of the fix. I think you meant, "none outside none" to "outside none none". 

Your JIRA comment on the tickets also look like a typo: `Changing value in the assertions from  "outside none none" to "outside none none" fix the issue.`

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:relevant

**Comment by john-co** [comment]
The SF test is failing due to the git commit message not starting with LRQA-xxx or other project...did you try running SF and checking the results?...

**Comment by john-co** [comment]
Test is expected to fail on CI since it still uses Chrome65. Please attach poshi report to tickets showing they locally pass on Chrome 86.

This is a temp fix, we'll need to find a longer term test fix since the difference between `none outside none` and `outside none none` isn't a true test requirement we necessarily care about.

**Comment by PablitoBonito** [comment]
@john-co  Should I split then? I put two tickets in one message because the fix was the same for both of them and in the same file. Also I saw PR where are fixed several tickets by one time. Like here: [pr#489](https://github.com/liferay-frontend/liferay-portal/pull/489)

Yes, I did the typos, thanks for pointing that. Already updated the description and will correct the tickets with putting there the Poshi report as well.

**Comment by john-co** [comment]
@PablitoBonito , I'm talking about the git commits, not the title of the PR. In the PR you linked they split the commits by their respective tickets, see https://github.com/liferay-frontend/liferay-portal/pull/489/commits

**Comment by PablitoBonito** [comment]
Got it, thank you @john-co 
So what I should do now?

**Comment by manoelcyreno** [comment]
@PablitoBonito , as far I understood, you need:

1. create one LSP ticket (testing type) to tell fix the two tickets mentioned in this PR ... and use this ticket in the title of your PR and commits. (not use the LRQA to it)

2. to split your commit in two ... 
the first commit to `LPS-?????? Fix assertion value outside none none to none outside none from the LRQA-63100`
the second commit to `LPS-?????? Fix assertion value outside none none to none outside none from the LRQA-63099`

I think is that.

**Comment by john-co** [comment]
I think what Manoel proposed is fine. Keep in mind the typos for the PR Title and git commit messages: `none outside none to outside none none`

**Comment by manoelcyreno** [comment]
ci:test:sf

**Comment by manoelcyreno** [comment]
ci:test:relevant

**Comment by manoelcyreno** [comment]
ci:test:frontend

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase [hunk 1]
```diff
@@ -127,7 +127,7 @@ definition {
 		AssertCssValue(
 			locator1 = "//ul[contains(@id,'controlMenu')]",
 			locator2 = "list-style",
-			value1 = "outside none none");
+			value1 = "none outside none");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase]: @PablitoBonito , you're reverting the fix on this test...

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase [hunk 2]
```diff
@@ -84,7 +84,7 @@ definition {
 		AssertCssValue(
 			locator1 = "//ul[contains(@id,'controlMenu')]",
 			locator2 = "list-style",
-			value1 = "none outside none");
+			value1 = "outside none none");
 
 		takeScreenshot();
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase:90]: LPS-63100 is only for RefreshWebContent test, you're making changes to RefreshHomePage in this commit... you need to split the changes off to a different commit.

**Comment by john-co** [comment]
Please also unquarantine the two tests now that these tests have been quarantined.

**Review by john-co — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase (hunk 2 above)
Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase:90]: Please ask @manoelcyreno  or @zhangyan0701 if need clarification on how to do this.

**Comment by PablitoBonito** [comment]
@manoelcyreno would like to ask you for clarification.
1. I understood that:
commit LRQA-63100 should have only fix for assertion for RefreshWebContent , notheng else.
LRQA-63099 should have only fix forfor assertion for RefreshHomePage , nothing else.

2. Question during chat with Summer: 
Why I was  asked to create a LPS ticket?
Was not enough use commit message that start with LRQA-xxx ?
Because now two commit messages start with the same LPS id but actually they are fixing two different LRQA tickets.

**Comment by manoelcyreno** [comment]
1. I understood that:
exactly.
IMHO, on it should be...
Fix the LRQA-63100, send a PR with the solution.
Fix the LRQA-63099, send other different PR with the solution.

2. Question during chat with Summer:
2.1 -> When I try to commit some LRQA I have some problems in the PR, for this reason, I told to create the LPS.
2.2 -> You are solving two problems (two different tickets), so the best practice recommends you create one commit per solution.
2.3 -> so, you need to change your commit message to reflect the reality ...

**Comment by john-co** [comment]
I discussed with Pawel and clarified the two commit expectation. We will use LPS-123675 for now since it's already created, but in the future I don't think it's necessary to create a LPS for two tickets. We can sync about this in a meeting together if needed @manoelcyreno.

**Comment by PablitoBonito** [comment]
ci:test:sf

**Comment by PablitoBonito** [comment]
ci:test:relevant

**Comment by john-co** [comment]
Please remove `https://...` from your commit messages to start with valid project name.

**Comment by john-co** [comment]
The commits need to be rebased, sent to a new PR, and unquarantine the two tests. I'll do this since Pawel will be out on vacation.

**Comment by john-co** [comment]
pushed to https://github.com/liferay-frontend/liferay-portal/pull/592
